### PR TITLE
CREST fleet endpoints

### PR DESCRIPTION
**Pull request by CarbonAlabel**
I finally got around to making some finishing touches to the docs I've been working on for a while and submitting a pull request.

I'm also using a new format for endpoint pages, which I would like to see applied to all other CREST endpoints. Best take a look at it :smile:

### CODE REVIEW

**Comment by timpanogos** [comment]
I'm all for putting endpoints under folder structures that are organized by functionality.

Looks good to me.

**Comment by timpanogos** [comment]
Can you supply at list one endpoint.md?

Also please list here in comments all of the available full path url/uri's dealing with fleet.

**Comment by timpanogos** [comment]
I hear some potential changes are ahead.
we should consider merging our good points and losing bad points, merging to one proposed template.
Should we do a side by side analysis or some such?

File: docs/crest/fleet/fleet_squads.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Fleet Squads
+
+## Fleet Squads
+
+The fleet squads collection lets allows you to create new squads.
```
Comment (OrthoLoess) [inline docs/crest/fleet/fleet_squads.md]: small typo here (extra word)

**Comment by OrthoLoess** [comment]
On first look, I like this proposed format. Have not yet compared properly with current, or with Chad's.

In the fleet_fleet page, it would be useful to explain where the link is obtained from and who can access the resources.

**Comment by timpanogos** [comment]
ccp has seen my temp .txt files so I will be deleting them.  It was mentioned that if a good list was available it would be an easy/quick fix after current priorities to fixup the EndpointCollections (soon to rename to RootEndpoint) to have a complete list.  I assume this could include fixups to the group/endpoint inconsistencies.    

I bring this up here as I have not seen these fleet endpoints in any of my RootEndpoint/Options work and I wonder how many others there might be that the community might know about but are not in this documentation at this time (nor in the RootEndpoint like fleet).

I also mention this here as we should start a cleaner document to present to ccp on all known issues with RootEndpoint which could effect our choice of naming/doc-structure.

**Comment by timpanogos** [comment]
@CarbonAlabel will you please propose a single endpoint.md as your "template" and I will stick to and start modifying TournamentCollection.md as mine.  I'd like to start moving some stuff I see as good from yours to mine.  Perhaps this would help us merge towards a singular.

**Comment by jonobrien** [comment]
#221 for tournaments I have documented thusfar

**Comment by timpanogos** [comment]
https://github.com/timpanogos/eveonline-third-party-documentation/blob/master/docs/crest/misc/tournament/tournamentCollection.md my current stab, bringing @CarbonAlabel 's routing/navigation concept down ... I liked this a lot

**Comment by CarbonAlabel** [comment]
Since opinions on the format proposed here seem to be mostly favourable, I'm going to open up an issue for deciding on a CREST endpoint documentation format, where we can discuss and further refine this format (or decide on another one), without the discussion being spread out over multiple PRs.

**Comment by timpanogos** [comment]
I do not see an issue .. I'd basically suggest just a couple things:

1. Title name ... make if the full access string name thus reflecting the full UID including version of the endpoint being documented.  i.e.

application/vnd.ccp.eve.Fleet*-v1+json

2. add the pseudo data structure in addition to the json sample.

3. In English sentences document the pseudo data structure and its use in navigation and/or other notes of interest about its use.  The sample should not need documentation.  I find bulleted lists that simply mirror the obvious from the data structure/samples as redundant.

**Comment by timpanogos** [comment]
In trying to gather all currently known endpoints in Tournament by looking at @OrthoLoess tournament PR, in combination to the static and realtime there are at least nine unique endpoints.

Now I'm back to the need of our documentation (and previous up thread concern) to closely follow the [group][endpoint] pattern where each group has a dedicated .md describing that group.  i.e. a tournamentGroup.md and a fleetGroup.md that then highlight documents that group and links all that groups endpoints.

**Comment by timpanogos** [comment]
@CarbonAlabel I do hope that my latest tournamentGroup.md suggestion highlights the importance of the titles of all endpoint.md's being the ccp given uid name/version.  I only have three hot links off of that group page, the other eight I lifted the navigation bullets from @OrthoLoess PR

**Comment by jonobrien** [comment]
you might run into issues with api versions for individual endpoints being much greater than others, if the offset in version numbers isn't an issue, that's fine with me, and resolving existing collisions is avoided as well.

**Pull request merged**